File: backend/services/post-rust-actix-web-fdm/src/common/api/invalid_page_size.rs
use crate::common::utils::error::*;

impl super::super::PageSize {
    pub(in crate::common) fn invalid_page_size() -> ErrorBody {
        ErrorBody {
            error: ErrBody {
                error: "INVALID_PAGE_SIZE".into(),
                reason: "page size is invalid".to_string(),
                message: "page size is invalid".to_string(),
            },
        }
    }

    pub(in crate::common) fn page_size_too_large() -> ErrorBody {
        ErrorBody {
            error: ErrBody {
                error: "PAGE_SIZE_TOO_LARGE".into(),
                reason: "page size is too large".to_string(),
                message: "page size is too large".to_string(),
            },
        }
    }
}

File: backend/services/post-ts-koa-mongo-fdm/src/common/api/invalid-page-size.ts
import { ErrorBody } from "../utils/error";

export const invalidPageSize: ErrorBody = {
    error: {
        error: "INVALID_PAGE_SIZE",
        reason: "page size is invalid",
        message: "page size is invalid",
    },
};

export const pageSizeTooLarge: ErrorBody = {
    error: {
        error: "PAGE_SIZE_TOO_LARGE",
        reason: "page size is too large",
        message: "page size is too large",
    },
};
